.github/workflows/code-checks.yml

```
@@ -37,4 +37,5 @@ jobs:
         pip list
 
     - name: Type check
-      run: mypy --install-types --non-interactive
+      working-directory: ./src
+      run: mypy -p pytorch_lightning --install-types --non-interactive --verbose --config-file ../pyproject.toml --warn-unused-configs
```

Verbose is for debugging mypy issues

And warn-unused-configs is already set in `pyproject.toml`

```suggestion
      run: mypy -p pytorch_lightning --install-types --non-interactive --config-file ../pyproject.toml
```